crates/extensions/c8y_mapper_ext/src/config.rs

```diff
@@ -41,7 +41,6 @@ const SUPPORTED_OPERATIONS_DIRECTORY: &str = "operations";
 pub struct C8yMapperConfig {
     pub device_id: String,
     pub device_topic_id: EntityTopicId,
-    pub device_type: String,
```

Comment:
Nice to see this property gone. This has been a source of troubles from the start.